org.flatpak.demo.json

```diff
@@ -0,0 +1,36 @@
+{
+    "app-id": "org.flatpak.demo",
```

Comment:
I assume that using this namespace is OK with @alexlarsson given the way development has taken place but I'd like to see an explicit OK from him

Reply:
Maybe what a more unique name also in case we have multiple demos? Like ElectronDemo, QtDemo, etc?

Reply:
Note there is `org.flathub.electron-sample-app` https://github.com/flathub/electron-sample-app

Reply:
What do you suggest then? Use org.flatpak.qtdemo or something like that?

Reply:
qtdemo makes sense to me. or qt-sample-app. Maybe worth checking with @allanday since I know he's been trying to make sure we're being vaguely coherent.

Reply:
The goal for the demo app is to demonstrate how to use Flatpak, rather than how to use a particular platform - we wanted to show generic Flatpak features that most apps will use. The fact that the app uses Qt is, at least theoretically, incidental.

Of course, some platforms *do* require their own tricks with Flatpak, which is why we have an Electron demo app. However, the point there is to show the deviations from standard practice.

Ultimately, it might boil down to a question of whether, from a Flatpak usage point of view, a Qt demo substantively differs from a GTK one, or a Freedesktop one. Is it a good representation of basic Flatpak usage?